config/locales/en.yml

```diff
@@ -568,7 +568,7 @@ en:
       meta_description: GOV.UK - The place to find government services and information - simpler, clearer, faster.
       more: More on GOV.UK
       popular_links_heading: Popular on GOV.UK
-      popular_links:
+      popular_links_a_z:
```

Comment:
Minor thing: I think we should undo this name change, and have this default set of links still called `popular_links`. So if 2nd line has to turn this AB test off urgently, there's minimal confusion about what the links used to be before the AB test was activated.